charts/rekor-tiles/values.yaml

```diff
@@ -0,0 +1,137 @@
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Automatically mount a ServiceAccount's API credentials?
+  automount: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+podLabels: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  ports:
+    - name: 3000-tcp
+      port: 80
+      protocol: TCP
+      targetPort: 3000
+    - name: 3001-tcp
+      port: 3001
+      protocol: TCP
+      targetPort: 3001
+    - name: 2112-tcp
+      port: 2112
+      protocol: TCP
+      targetPort: 2112
+
+ingress:
+  enabled: true
+  className: "gce"
+  annotations: {}
+  hosts:
+    - path: /
+      host: root
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+  # staticGlobalIP: lb-ext-ip
+  # frontendConfigSpec:   # https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration#configuring_ingress_features_through_frontendconfig_parameters
+  #   sslPolicy: rekor-tiles-ssl-policy
+  #   redirectToHttps:
+  #     enabled: true
+  # backendConfigSpec:   # https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-configuration#configuring_ingress_features_through_backendconfig_parameters
+  #   securityPolicy:
+  #     name: rekor-tiles-security-policy
+  #   logging:
+  #     enable: true
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+livenessProbe:
+  httpGet:
+    path: /healthz
+    port: 3000
+readinessProbe:
+  httpGet:
+    path: /healthz
+    port: 3000
+
+nodeSelector: {}
+  # iam.gke.io/gke-metadata-server-enabled: "true"   # support workload identity
+
+tolerations: []
+
+affinity: {}
+
+# Server arguments
+antispam: {}
+tesseraLivecycle: {}
+gcp: {}
+grpc:
+  port: "3001"
+hostname: "localhost"
+http:
+  port: "3000"
+  metricsPort: "2112"
+server: {}
+readOnly: false
+signer: {}
+  # file:
+  #   path: /pki/signer.pem
+  #   withPassword: false
+  #   secret:
+  #     name: "signing-key"
+  #     key: signing-key
+  #     mountPath: /pki
+  #     mountSubPath: signer.pem
+  # kms:
+  #   hash: sha256
+  #   key: gcpkms://keyname
+  # tink:
+  #   kekURI: gcp-kms://keyname
+  #   configMapName: tink-config
+  #   keyset: ""
+extraArgs: []
```

Comment:
perhaps a nit but most (if not all) charts in this repo nest all server arguments under `server` instead of having them as top-level... for consistency it might be nice to do that but certainly open to comments

Reply:
I veered away from that on purpose but I don't feel that strongly about it. The reason rekor does it is that it has multiple deployments and each main section (`server`, `mysql`, and `redis`) are configuring that specific deployment https://github.com/sigstore/helm-charts/tree/main/charts/rekor/templates

Here we're just configuring one thing. I guess I could nest them preemptively in case we add more stuff?

Reply:
We will eventually support mysql, so I could see us using the same pattern.

Reply:
Done